jenkins/jobs/bml_integration_tests.pipeline

```diff
@@ -32,30 +33,21 @@ pipeline {
   agent { label 'metal3ci-bml-jenkins-worker' }
   environment {
     METAL3_CI_USER="metal3ci"
-    REPO_ORG = "${PROJECT_REPO_ORG}"
-    REPO_NAME = "${PROJECT_REPO_NAME}"
-    UPDATED_REPO = "${ghprbAuthorRepoGitUrl}"
-    REPO_BRANCH = "${ghprbTargetBranch}"
-    UPDATED_BRANCH = "${ghprbActualCommit}"
-    TEST_EXECUTER_VM_NAME = "${VM_NAME}"
+    REPO_ORG = "${env.REPO_OWNER}"
+    REPO_NAME = "${env.REPO_NAME}"
+    UPDATED_REPO = "${UPDATED_REPO}"
+    REPO_BRANCH = "${env.PULL_BASE_REF}"
+    UPDATED_BRANCH = "${env.PULL_PULL_SHA}"
     BUILD_TAG = "${env.BUILD_TAG}"
-    PR_ID = "${ghprbPullId}"
+    PR_ID = "${env.PULL_NUMBER}"
     IMAGE_OS = "${IMAGE_OS}"
-    CAPI_VERSION = "${CAPI_VERSION}"
-    CAPM3_VERSION = "${CAPM3_VERSION}"
     CAPM3RELEASEBRANCH = "${capm3_release_branch}"
     BMORELEASEBRANCH = "${bmo_release_branch}"
     TARGET_NODE_MEMORY = "${TARGET_NODE_MEMORY}"
-    NUM_NODES="${NUM_NODES}"
-    GINKGO_FOCUS="${GINKGO_FOCUS}"
-    GINKGO_SKIP="${GINKGO_SKIP}"
-    EPHEMERAL_TEST="${EPHEMERAL_TEST}"
-    TESTS_FOR="${TESTS_FOR}"
-    BARE_METAL_LAB=true
```

Comment:
Are all of these L54-L58 also obsolete and supposed to be removed?

Reply:
yes, first three is related to e2e tests. And the BARE_METAL_LAB not needed bcs we hve now separate pipeline for each type of tests.